sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -251,7 +247,7 @@ object SQLConf {
       "of org.apache.parquet.hadoop.ParquetOutputCommitter.")
     .internal()
     .stringConf
-    .createWithDefault(classOf[ParquetOutputCommitter].getName)
+    .createWithDefault("org.apache.parquet.hadoop.ParquetOutputCommitter")
```

Comment:
It is a bit weird to me that we refer to sql/core configs in catalyst. Shouldn't we just move the dynamic configuration and the catalyst only configurations to CatalystConf and make SQLConf extend this?